package.json

```diff
@@ -9,13 +9,14 @@
 		"build": "vite build",
 		"preview": "vite preview",
 		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
-		"test": "vitest",
+		"test": "vitest run && node scripts/validateData.js",
```

Comment:
Do we test (as Unit testing, functional testing) the data, or we lint (as check if it's matching our quality criteria) the data ?

For me, it's the latter, but I'm open to discussion